### DISCUSSION TITLE
KeyError: 'a0d7b954_thelounge' - Add-on never installed

### DISCUSSION THREAD

**Discussion by 7wells** [hassio-addons/app-thelounge#141] [General]
Hello! In the HA Supervisor log I see a row about the Lounge add-on (self-hosted web IRC client) which I never installed. The row is this:
`KeyError: 'a0d7b954_thelounge'`
I am concerned and worried why a log in HA refers to an add-on I never installed. Is this a security threat? 🧐 Thanks for sharing your thoughts on this.

**Comment by frenck** [comment]
Could you share a bigger log?

**Reply by 7wells** [reply]
My bad that I did not backup the log, as it's gone after I setup HA all new again. I am really sorry that I might have wasted your time. 🫢 Please accept my sincere apologies.
